py3status/modules/file_status.py

```diff
@@ -68,16 +92,40 @@ def file_status(self):
                 'cached_until': self.py3.CACHE_FOREVER,
             }
 
-        if exists(self.path):
-            icon = self.icon_available
+        # init data
+        file_status_data = {}
+        # expand glob from paths
+        paths = list(map(glob, self.path))
+        # merge list of paths
+        paths = [x for x in chain.from_iterable(paths)]
+
+        # fill data
+        file_status_data['icon'] = self.icon_unavailable
+        color = self.py3.COLOR_BAD
+
+        if paths:
+            file_status_data['icon'] = self.icon_available
             color = self.py3.COLOR_GOOD
-        else:
-            icon = self.icon_unavailable
-            color = self.py3.COLOR_BAD
+
+        # format paths
+        if self.format_path:
+            format_path = {}
+
+            format_path_separator = self.py3.safe_format(
+                self.format_path_separator)
+
+            format_path['basename'] = self.py3.composite_join(
+                format_path_separator, map(basename, paths))
+
+            format_path['full'] = self.py3.composite_join(
```

Comment:
Due to glob, there can be many things in the list. We could be a bit more efficient by using  `get_placeholders_list`-in-`post_config_hook` list for `format_path` and find out which placeholder we want to join... rather than always joining them both.

Look `composite_join` in `composite.py`. It makes a composite obj, go thru list, append separator, append item, etc... which could always be skipped on every interval instead. This is an optional nit.